packages/ember/tests/routing/basic_test.js

```diff
@@ -340,11 +340,11 @@ test('Renders correct view with slash notation', function() {
 
   bootApplication();
 
-  equal(jQuery('p:contains(Home/Page)', '#qunit-fixture').length, 1, 'The homepage template was rendered');
+  equal(jQuery('p:contains(put some respeck on it)', '#qunit-fixture').length, 1, 'The homepage template was rendered');
 });
 
-test('Renders the view given in the view option', function() {
-  setTemplate('home', compile('<p>{{view.name}}</p>'));
+QUnit.test('Renders the view given in the view option', function() {
```

Comment:
Canary is dropping support for views, and thus also the `view` option to `render`. This test can be removed.